### ISSUE TITLE
Will the code be published?

### ISSUE THREAD

**Issue by TomArrow** [PeterWang512/FALdetector#2]
Was just wondering if this code will be published? The project page linked here for the code, but the repository is empty.

**Comment by PeterWang512** [comment]
Yes, the testing code is now released, and we will release our training code as well in the future. Thank you!

**Comment by AOTY-szy** [comment]
How great your work, is there any plan to still release the training code?

**Comment by PeterWang512** [comment]
Sorry about the plan change, but we cancelled the plan for training code release due to licensing issues with the training dataset.
